modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/ObjectEntryOpenAPIContributor.java

```diff
@@ -126,28 +133,66 @@ else if (key.contains("objectRelationshipName")) {
 								relatedSchemaName);
 						}
 
-						_addObjectRelationshipPathItem(
-							key, objectRelationship, paths, relatedSchemaName);
+						if (!(Objects.equals(
```

Comment:
I think it would be better if you include the cases in which the path item should be included, for example:

```
if ((objectRelationship.getObjectDefinitionId1() ==
		_objectDefinition.getObjectDefinitionId()) &&
	(Objects.equals(
		ObjectRelationshipConstants.TYPE_ONE_TO_MANY,
		objectRelationship.getType()) ||
	 Objects.equals(
		 ObjectRelationshipConstants.TYPE_MANY_TO_MANY,
		 objectRelationship.getType()))) {

	_addObjectRelationshipPathItem(
		key, objectRelationship, paths,
		relatedSchemaName);
}
```